librz/hash/sha2.c

```diff
@@ -455,9 +455,6 @@ void SHA256_Transform(RZ_SHA256_CTX *context, const ut32 *data) {
 	context->state[5] += f;
 	context->state[6] += g;
 	context->state[7] += h;
-
-	/* Clean up */
```

Comment:
At line 380 I see similar pattern. Is that dead as well?

Reply:
i removed all useless clean up comment